src/fightPage/buttonsEventListeners.ts

```diff
@@ -98,8 +98,6 @@ const magicFunction = (
       createActivePlayer(gameHandler);
     }
     setTimeout(() => {
```

Comment:
Dont't delete genateAttackButtons, otherwise the attack types won't change when the pokemon is changed

Reply:
how is it possible? 😅